src/views/Login.jsx

```diff
@@ -14,13 +14,13 @@ const Login = () => {
         e.preventDefault();
 
         try {
-            const response = await axios.post('https://reqres.in/api/login', {
-                email,
-                password,
-            });
+            const token = await login({ email, password });
```

Comment:
proposal: move this logic to an aux service/module/file ; `login-service.jsx` i a file (and a module) which is going to contain this logic (we can add later, the cache, the try-catching, maybe some logging services, and might be easier to test)

the golden rule is: this might have entity by its own? a.k.a will i be able to test this piece of code isolated in a unit test? -> different file